### PR TITLE
Add RelationshipGroup struct

### PR DESCRIPTION
**Pull request by kagikn**

### CODE REVIEW

**Comment by Nacorpio** [comment]
It's "relationship", not "relationShip".

**Comment by kagikn** [comment]
Oh, will fix this class name.

File: ScriptHookVDotNet.vcxproj [hunk 1]
```diff
@@ -64,6 +64,7 @@
   </ItemGroup>
   <!-- Core source files -->
   <ItemGroup>
+    <ClCompile Include="..\scripthookvdotnet_temp\rela\RelationshipGroup.cpp" />
```
Comment (crosire) [inline ScriptHookVDotNet.vcxproj]: Eh?

**Comment by kagikn** [comment]
I fixed the references.

**Comment by Fireboyd78** [comment]
Do a find and replace all operation for every string matching "relationShipGroup" and you should be good to go!

**Pull request merged**